zjit/src/hir.rs

```diff
@@ -652,6 +657,8 @@ pub enum Insn {
         return_type: Option<Type>,  // None for unannotated builtins
     },
 
+    /// Set up frame and remember the address as a JIT entry if insn_idx is Some
+    EntryPoint { jit_entry: bool },
```

Comment:
There's no `insn_idx` in this struct-like enum so the comment is a little hard to understand / misleading

Reply:
Yeah sorry, it was an outdated comment https://github.com/ruby/ruby/pull/14643/commits/adeb771c9895f3eac4b6b8e5e852015cd22edd3c